common/rfb/EncodeManager.h

```diff
@@ -57,7 +57,7 @@ namespace rfb {
 
     void writeUpdate(const UpdateInfo& ui, const PixelBuffer* pb,
                      const RenderedCursor* renderedCursor,
-                     size_t maxUpdateSize);
+                     size_t maxUpdateSize = 2000);
```

Comment:
Update the tests instead. We don't want to overlook this parameter as it will be crucial to get right.